dlt/destinations/qdrant/qdrant_adapter.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any
+
+from dlt.common.schema.typing import TColumnNames, TTableSchemaColumns
+from dlt.extract.decorators import resource as make_resource
+from dlt.extract.source import DltResource
+
+VECTORIZE_HINT = "x-embedding-field"
```

Comment:
To make it clear where the hint is coming from I suggest to use a different naming: `x-qdrant-embed`.

Reply:
https://github.com/dlt-hub/dlt/pull/724/commits/3c25f71a36f40d7f44d737cac140afef1e1571b3